src/test/java/org/isf/medicalstock/Tests.java

```diff
@@ -782,6 +783,26 @@ void testMgrGetLotsByMedicalNull(boolean in, boolean out, boolean toward) throws
 		assertThat(movStockInsertingManager.getLotByMedical(null, true)).isEmpty();
 	}
 
+	@ParameterizedTest(name = "Test with AUTOMATICLOT_IN={0}, AUTOMATICLOT_OUT={1}, AUTOMATICLOTWARD_TOWARD={2}")
+	@MethodSource("automaticlot")
+	public void testMgrUpdateLots() throws Exception {
```

Comment:
```suggestion
	void testMgrUpdateLots() throws Exception {
```
With Junit 5 the standard is not to make test methods public; see all the other methods in this file

Reply:
Right! Thanks!